db/migrate/20200428014005_correct_topic_user_bookmarked_boolean.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CorrectTopicUserBookmarkedBoolean < ActiveRecord::Migration[6.0]
+  def up
+    # if the relation exists then we set to bookmarked because
+    # at least 1 bookmark for the user + topic exists
+    DB.exec(
+      <<~SQL
+        UPDATE topic_users SET bookmarked = true
+        FROM bookmarks AS b
+        WHERE topic_users.bookmarked IS false AND topic_users.topic_id = b.topic_id AND topic_users.user_id = b.user_id
```

Comment:
`IS false` feels a bit weird ... `NOT topic_users.bookmarked` ... for the `true` case `where topic_users.bookmarked`